### DISCUSSION TITLE
Is still support php?

### DISCUSSION THREAD

**Discussion by xxm404** [grafana/pyroscope#2325] [Q&A] [answered]
In pyroscope demo https://github.com/grafana/pyroscope/tree/next/examples/php i found php demo, but in doc page https://grafana.com/docs/pyroscope/latest/configure-client/#choosing-the-grafana-agent-or-pyroscope-sdk-to-send-profiles php missing, so pyroscope v1.0 is still support php?

**Comment by EdanBrooke** [comment]
I am looking for the same information. All I've found so far is this mention in the Pyroscope v1.0 upgrade guide that states support forcertain subcommands including the `exec` subcommand has been dropped. The existing documentation tells you to use phpspy with these now deprecated commands so it looks like Grafana don't care about profiling of PHP apps anymore.

https://grafana.com/docs/pyroscope/latest/upgrade-guide/#dropping-support-for-certain-subcommands

**Comment by cyriltovena** [comment, marked as answer]
We don't have support anymore for the client and the `exec`, but you can still use the old instrumentation and send the data to the new pyroscope without any issue.

We do care about php, we have plans to  bring it back, but since the old one still works, it's not a priority.

**Reply by abdallah** [reply]
which old instrumentation works with the new pyroscope? can you point to any documentation that works?

**Reply by korniltsev** [reply]
You can try using old pyroscope exec from og branch, but it is not efficient and not supported.

**Comment by everythings-gonna-be-alright** [comment]
Hi everyone. I wrote an adapter.
It is easy to use and can solve the current issue.

https://github.com/everythings-gonna-be-alright/phpScope

```
curl -sSL https://raw.githubusercontent.com/everythings-gonna-be-alright/phpScope/refs/heads/main/docker-compose.yaml | docker compose -f - up -d
```
